File: URI/1_Iniciante/1040/1040.py
import sys

nota_1, nota_2, nota_3, nota_4 = map(float, sys.stdin.readline().split(' '))

media_s = nota_1*2 + nota_2*3 + nota_3*4 + nota_4*1
media = media_s / 10

print("Media: {:.1f}" .format(media))
if media > 7.0:
    print("Aluno aprovado.")
elif media < 5.0:
    print("Aluno reprovado.")
elif media >= 5.0 or media <= 6.9:
    print("Aluno em exame.")
    nota_exame = float(input("Nota do exame: "))
    print(nota_exame)
    media_exame = (media + nota_exame)/2.0
    if media_exame > 5.0:
        print("Aluno aprovado.")
    elif media_exame <= 4.9:
        print("Aluno reprovado.")

    print("Media final: {}" .format(media_exame))

File: URI/1_Iniciante/1040/1040.c
#include <stdio.h>

int main(int argc, char *argv[]) {
    float nota1, nota2, nota3, nota4;
    float media, media_final;
    float nota_exame;

    scanf("%f %f %f %f", &nota1, &nota2, &nota3, &nota4);
    media = (nota1*2 + nota2*3 + nota3*4 + nota4*1)/10;

    printf("Media: %.1f\n", media);

    if (media >= 7.0) {
        printf("Aluno aprovado.\n");
    } else if (media < 5.0) {
        printf("Aluno reprovado.\n");
    } else if (media >= 5.0 || media <= 6.9) {
        printf("Aluno em exame.\n");
        printf("Nota do exame: ");
        scanf("%f", &nota_exame);
        printf("%.1f\n", nota_exame);

        media_final = (media + nota_exame)/2.0;
        if (media_final >= 5.0) {
            printf("Aluno aprovado.\n");
        } else if (media_final <= 4.9) {
            printf("Aluno reprovado.\n");
        }
        printf("Media final: %.1f\n", media_final);
    }
    return 0;
}
